### DISCUSSION TITLE
2021/6/4放送予定

### DISCUSSION THREAD

**Discussion by kkamegawa** [tfsugjp/devblogradio#1] [General]
# 2021/6/4 放送予定内容

[devblogラジオ vol.17 ～Buildを振り返る特別編！～](https://devblog.connpass.com/event/214182/)

お題(予定)
- [Build consistent hybrid and multicloud applications with Azure Arc](https://mybuild.microsoft.com/sessions/bafa8204-25b2-44c1-86a0-2c852c1e3794?source=sessions)
- やってみた？
- 使いどころどうだろう
- Azure Arcの可能性

**Comment by tsubakimoto** [comment]
Hello world.

**Comment by kkamegawa** [comment]
@MasayukiOzawa がやってみた話をしてくれるので、招待する
